main.mustache

```diff
@@ -1,15 +1,15 @@
 # wordle-spoiler
 
 <details>
-  <summary>Click to reveal <b>today's</b> word!</summary>
+  <summary>Click to reveal <b>tomorrow's</b> word!</summary>
```

Comment:
Similarly, changing the order of the words feels out-of-scope for this PR. For single-ness of concern, let's stick to only updating the solution word list here. Please revert.